src/containers/PageConstructor/components/ConstructorItem/ConstructorItem.tsx

```diff
@@ -3,10 +3,16 @@ import React, {useContext} from 'react';
 import {BlockIdContext} from '../../../../context/blockIdContext';
 import {InnerContext} from '../../../../context/innerContext';
 import {BlockDecoration} from '../../../../customization/BlockDecoration';
-import {BlockType, ConstructorItem as ConstructorItemType, WithChildren} from '../../../../models';
+import {
+    BlockType,
+    // ConstructorItem as ConstructorItemType,
+    WithChildren,
+    // eslint-disable-next-line sort-imports
```

Comment:
Why?

Reply:
fix)) I chose the wrong solution to the eslint problem from the IDE